pinot-core/src/test/java/org/apache/pinot/queries/NativeAndLuceneComparisonTest.java

```diff
@@ -219,38 +217,41 @@ private void buildNativeTextIndexSegment()
 
   private ImmutableSegment loadLuceneSegment()
       throws Exception {
-    IndexLoadingConfig indexLoadingConfig = new IndexLoadingConfig();
-    Set<String> textIndexCols = new HashSet<>();
-    textIndexCols.add(QUOTES_COL_LUCENE);
-    textIndexCols.add(QUOTES_COL_LUCENE_MV);
-    indexLoadingConfig.setTextIndexColumns(textIndexCols);
-    Set<String> invertedIndexCols = new HashSet<>();
-    invertedIndexCols.add(QUOTES_COL_LUCENE);
-    invertedIndexCols.add(QUOTES_COL_LUCENE_MV);
-    indexLoadingConfig.setInvertedIndexColumns(invertedIndexCols);
+    List<FieldConfig> fieldConfigs = new ArrayList<>();
```

Comment:
(minor) Similarly here, we may use `List.of()` and `Map.of()` for concise. Same for other places